src/chemistry/mozart/mo_chm_diags.F90

```diff
@@ -748,8 +774,50 @@ subroutine chm_diags( lchnk, ncol, vmr, mmr, rxt_rates, invariants, depvel, depf
           call outfld('DO3CHM_LMS',do3chm_lms(:ncol), ncol, lchnk )
        end if
 !
-    enddo
+    enddo ! end loop from m=1,gas_pcnst
+
+    ! Generate compund sum
 
+    ! so4_a1,so4_a2,so4_a3
+    ! bc_a1,bc_a4
+    ! pom_a1,pom_a4
+    ! ncl_a1,ncl_a2,ncl_a3
+    ! soa_a1,soa_a2
+    ! num_a1,num_a2,num_a3,num_a4
```

Comment:
I think it would be nice if each of these comments was at the end of a line below (e.g., line 781 would be at the end of line 788).